Enigmatry.Entry.CodeGeneration/Templates/HtmlHelperExtensions/Angular/AngularFormlyHtmlHelperExtensions.cs

```diff
@@ -31,6 +31,18 @@ public static IHtmlContent GroupCssClass(this IHtmlHelper html, FormControlGroup
         return html.Raw($"fieldGroupClassName: `{classNameValue}`,\r\n");
     }
 
+    public static IHtmlContent DefaultValue(this IHtmlHelper html, FormControl control)
+    {
+        if (control.DefaultValue == null)
+        {
+            return html.Raw("");
+        }
+
+        return control.FormlyType == "checkbox"
```

Comment:
@msljivic , not sure if we have better way to compare formlyTypes? Should I introduce a class with constants maybe, or we already have it (could find it)?